spec/models/user_spec.rb

```diff
@@ -104,6 +118,29 @@
     end
   end
 
+  describe "#core_induction_programme" do
+    it "is expected to return mentor cip for mentor users" do
+      user = create(:user, :mentor)
+      cip = create(:core_induction_programme)
+      user.mentor_profile.core_induction_programme = cip
+
+      expect(user.core_induction_programme).to be cip
+    end
+
+    it "is expected to return ect cip for ect users" do
+      user = create(:user, :early_career_teacher)
+      cip = create(:core_induction_programme)
+      user.early_career_teacher_profile.core_induction_programme = cip
+
+      expect(user.core_induction_programme).to be cip
+    end
+
+    it "is expected to not blow up when no cip" do
```

Comment:
`to return nil` please, `blow up` can mean 'become very expensive' :P

Reply:
booooooo